### ISSUE TITLE
Fibaro FGD212 switch doesn't react when controlled from domoticz 

### ISSUE THREAD

**Issue by pioupiou92** [domoticz/domoticz#6192]
### Issue Guidelines

Before opening a new issue, please review the following:
* The Github Issues is for Domoticz CODE Bugs only. Please do not use this for general questions, howto, or this is not working conversations. If you have a question, please post to our forum at https://www.domoticz.com/forum
* Please use the search feature to see if your issues have been raised or addressed first.
* First try to update to the latest beta version
* Clear your browser cache + AppCache (two different things!!)
* Always state your version and hardware platform when reporting code related issues
* Often we will require Log Files and Screenshots to help diagnose the issue.

Hi,
Fibaro FGD212 module properly seen in Z-Wave JS UI. It is visible in MQTT Explore thru discovery. Also properly discovered by domoticz MQTT Auto Discovery Client Gateway.
But, while actionning the dimmer light or switching it on/off nothing happens.

It is working OK with domoticz OpenZwave controller. Also works OK when actionned by the physical switch connected to the module.

Topic and value attached.
Zwave protocol version 4.24, chip firmware 3.5, chip hardware version 2.

Any idea of what's going wrong?

Kind regards,
Pascal

Topic: homeassistant/light/nodeID_37/dimmer_1/config
payload:`{"command_topic":"zwave/nodeID_37/38/1/targetValue/set","state_topic":"zwave/nodeID_37/38/1/currentValue","state_value_template":"{{ \"OFF\" if value_json.value == 0 else \"ON\" }}","brightness_command_topic":"zwave/nodeID_37/38/1/targetValue/set","brightness_scale":99,"brightness_state_topic":"zwave/nodeID_37/38/1/currentValue","brightness_value_template":"{{ value_json.value }}","on_command_type":"brightness","supported_color_modes":["brightness"],"availability":[{"payload_available":"true","payload_not_available":"false","topic":"zwave/nodeID_37/status","value_template":"{{'true' if value_json.value else 'false'}}"},{"topic":"zwave/_CLIENTS/ZWAVE_GATEWAY-zwave-js-ui/status","value_template":"{{'online' if value_json.value else 'offline'}}"},{"payload_available":"true","payload_not_available":"false","topic":"zwave/driver/status"}],"availability_mode":"all","device":{"identifiers":["zwavejs2mqtt_0xedfc8921_node37"],"manufacturer":"Fibargroup","model":"Dimmer 2 (FGD212)","name":"nodeID_37","sw_version":"3.5"},"name":"nodeID_37_dimmer_1","unique_id":"zwavejs2mqtt_0xedfc8921_37-38-1-currentValue"}
`

**Comment by gizmocuz** [comment]
Could it be that there are more lights/switches created for this node/device, and that you are using the wrong one?
Some devices have also a child lock, and other switches created

Could you tell what is send to 'zwave/nodeID_37/38/1/currentValue' when you set the dimmer to lets say 50%, it should send

{ "value": 50 } (instead of 50 it could be another number)

what get's send when you operate the dimmer in zwavejs2mqtt to this topic?

**Comment by pioupiou92** [comment]
Thank you for your quick answer. You make me think twice !

During initial inclusion, I didn’t set a name so it came up with generic name similar to NodeID_37. I renamed it afterwards. This is probably the root cause.
Today I exclude it then re-include it by setting a name and everything went as expected. Domoticz is controlling the module perfectly.

Before doing this, I removed the module name on Zwave js ui but it didn’t fix the issue.

Domoticz may have « received » the module with the generic name and was not able to control it after it was renamed.

Regards
Pascal
